bob-backend/src/pearl/group.rs

```diff
@@ -204,7 +214,10 @@ impl Group {
         data: &BobData,
         timestamp_config: StartTimestampConfig,
     ) -> Result<(), Error> {
-        let _reinit_lock = self.reinit_lock.try_read().map_err(|_| Error::holder_temporary_unavailable())?;
+        let _reinit_lock = self
+            .reinit_lock
+            .try_read()
+            .map_err(|_| Error::holder_temporary_unavailable())?;
```

Comment:
Please, revert this formatting change.
Also, please, disable auto formatting on the whole file. It generates many irrelevant changes